lambda/handlers/intentHandler.js

```
@@ -730,27 +731,253 @@ const HadithIntentHandler = {
+    const requestAttributes =
+      handlerInput.attributesManager.getRequestAttributes();
+    let speakOutput = "";
+    switch (code) {
+      case 200:
+        return handlerInput.responseBuilder
+          .speak(requestAttributes.t("routineCreatedPrompt") + requestAttributes.t("doYouNeedAnythingElsePrompt"))
+          .withShouldEndSession(false)
           .getResponse();
+      case 204:
+        const error =
+          handlerInput.requestEnvelope.request?.cause?.result?.offerAutomationResponse?.reason;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)